### PR TITLE
Monitoring: Added disk available capacity monitoring

### PR DESCRIPTION
**Pull request by retime123**
## Purpose of the pull request
This method is added because the available disk capacity cannot be monitored  
Previous   https://github.com/apache/dolphinscheduler/pull/10030

## Brief change log

Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java
Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/HeartBeat.java
Change dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/OSUtils.java
Change dolphinscheduler-ui/src/views/monitor/servers/master/index.tsx
Change dolphinscheduler-ui/src/views/monitor/servers/worker/index.tsx
Change dolphinscheduler-ui/src/locales/modules/en_US.ts
Change dolphinscheduler-ui/src/locales/modules/zh_CN.ts

unit testing
Change dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/os/OSUtilsTest.java
Change dolphinscheduler-common/src/test/java/org/apache/dolphinscheduler/common/utils/HeartBeatTest.java

## Verify this pull request

I have test local

image

### CODE REVIEW

**Comment by SbloodyS** [comment]
I had restart the failed CI.

**Comment by SbloodyS** [comment]
@retime123 Please note that WorkerGroup E2E tests failed.

**Comment by retime123** [comment]
This WorkerGroup E2E, can you change it?

**Comment by SbloodyS** [comment]
> This WorkerGroup E2E, can you change it?

This means you have a bug in your code.

**Review by CalvinKirs — COMMENTED** [review]
I don't know if it's correct in the K8s environment, it's better to come to someone who knows here to review.

File: dolphinscheduler-ui/src/locales/modules/en_US.ts [hunk 1]
```diff
@@ -148,6 +148,7 @@ const monitor = {
   master: {
     cpu_usage: 'CPU Usage',
     memory_usage: 'Memory Usage',
+    disk_available: 'disk Available',
```
Comment (Amy0104) [inline dolphinscheduler-ui/src/locales/modules/en_US.ts]: The first letter should be capitalized.

File: dolphinscheduler-ui/src/locales/modules/en_US.ts [hunk 2]
```diff
@@ -161,6 +162,7 @@ const monitor = {
   worker: {
     cpu_usage: 'CPU Usage',
     memory_usage: 'Memory Usage',
+    disk_available: 'disk Available',
```
Comment (Amy0104) [inline dolphinscheduler-ui/src/locales/modules/en_US.ts]: Same as above.

File: dolphinscheduler-ui/src/locales/modules/en_US.ts (hunk 1 above)
Comment (labbomb) [reply dolphinscheduler-ui/src/locales/modules/en_US.ts]: +1

**Review by labbomb — APPROVED** [review]
LGTM

File: dolphinscheduler-ui/src/locales/modules/en_US.ts (hunk 2 above)
Comment (retime123) [reply dolphinscheduler-ui/src/locales/modules/en_US.ts]: Ok thank you

File: dolphinscheduler-ui/src/locales/modules/en_US.ts (hunk 1 above)
Comment (retime123) [reply dolphinscheduler-ui/src/locales/modules/en_US.ts]: Ok thank you

**Comment by retime123** [comment]
By the way: will version 2.0 replace the upgrade?

**Comment by retime123** [comment]
E2E  That's a little tricky

**Pull request closed without merging**